packages/parser/test/specs/large-file-memory-leak/large-file-memory-leak.test.ts

```diff
@@ -9,16 +8,20 @@ describe('Large file memory leak protection', { timeout: 20000 }, () => {
     ['cloudflare spec (stringified to a single line)', 'cloudflare-stringified.json'],
     ['cloudflare spec', 'cloudflare.json'],
   ])('%s', async (name, file) => {
-    try {
-      await validate(relativePath(`specs/large-file-memory-leak/${file}`));
+    const res = await validate(relativePath(`specs/large-file-memory-leak/${file}`));
+    if (res.valid === true) {
```

Comment:
this if-block seems redundant

Reply:
it's for type narrowing. i could move this over to a snapshot but with 20 errors we're asserting here that'd be a **huge** snapshot